src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2568,12 +2571,21 @@ def aks_update(cmd, client, resource_group_name, name,
                         subscription_id=subscription_id,
                         detach=True)
 
+    if uptime_sla and no_uptime_sla:
```

Comment:
how about enable no uptime sla flag by setting uptime-sla to false? it's usual to have 2 params for a bool type config

Reply:
That's a PM decision to use a separate switch. It's consistent with --enable-cluster-scaler and --disable-cluster-scaler